push_hono_images.sh

```diff
@@ -34,23 +35,45 @@ NATIVE_IMAGES="hono-adapter-amqp-native \
         hono-adapter-mqtt-native \
         hono-adapter-sigfox-native \
         hono-service-auth-native \
+        hono-service-command-router-infinispan-native \
         hono-service-command-router-native \
         hono-service-device-registry-jdbc-native \
         hono-service-device-registry-mongodb-native"
 
 ME=$(basename "$0")
 echo "called as $ME"
 
+COMMAND_ROUTER_IMAGE_LEGACY_BASE="hono-service-command-router"
+COMMAND_ROUTER_IMAGE_INFINISPAN_BASE="${COMMAND_ROUTER_IMAGE_LEGACY_BASE}-infinispan"
```

Comment:
FMPOV we should move these declarations to before the `IMAGES` declaration and use the variables in the `IMAGES` list already.

Reply:
Will do. But what are your thoughts on the TODO I added to the script... the one that exits if the tagging isn't successful? Is that maybe overkill? Should I remove it?

Reply:
No, please keep it. You should also add the same logic to the part where all the other images are being tagged if another repo has been set.

Reply:
Allright. I've pushed that change. The caveat of course being that the script may go through pushing some of the images but then exiting if one of the image does not exist. The only way to get around that would be to split the for loop into two, where one loop does the tagging while the other one does the pushing. That would result in a bunch of duplicate logic though so I didn't go there (it could possibly be mitigated to some extent by moving stuff into functions). If you want me to go there just let me know :)

Reply:
I'd say this is more a question of how much (more) time you are willing to spend 😁

Reply:
Allright. I split the script up into two for loops.

The first loop goes through the images and either re-tags them or runs `docker inspect` to make sure they exist. In this loop I collect all images that do exist into an array `IMAGES_TO_PUSH`.

The second loop then goes through that array and pushes them.

Things could be made even more elaborate but I think this is where I'll draw the line :)

There is one known caveat (which IMO is minor, and also existed before) that the script can re-tag some images and then hit an image that doesn't exist and exit. This would mean that part of the images have been (locally) re-tagged (but none of the images were then pushed).

But as I said, I think this is as good as it gets without complicating the script much further.

Reply:
This is great :+1: